File: custom-addons/helpdesk_sms/models/helpdesk_stage.py
# -*- coding: utf-8 -*-
# Part of Odoo. See LICENSE file for full copyright and licensing details.

from odoo import fields, models


class HelpdeskStage(models.Model):
    _inherit = "helpdesk.stage"

    sms_template_id = fields.Many2one('sms.template', string="SMS Template",
        domain=[('model', '=', 'helpdesk.ticket')], help="SMS automatically sent to the customer when the ticket reaches this stage.")

File: enterprise/helpdesk/static/tests/mock_server/mock_models/helpdesk_stage.js
import { fields, models } from "@web/../tests/web_test_helpers";

export class HelpdeskStage extends models.Model {
    _name = "helpdesk.stage";

    name = fields.Char({ string: "Name" });

    _records = [{ name: "Stage 1" }, { name: "Stage 2" }, { name: "Stage 3" }];
}
